### DISCUSSION TITLE
Taro源码解析（3.5.4） | One Hacker

### DISCUSSION THREAD

**Discussion by giscus** [yunfengsa/yunfengsa.github.io#47] [Announcements]
# Taro源码解析（3.5.4） | One Hacker

对Taro3.x的认知一直停留在强运行时，知道其大致的原理是通过在运行时进行内容的渲染，类似重新编写react-dom的运行时，从而可以完整使用React生态内容，达到在小程序测渲染的目的。作为一个非

https://www.onehacker.top/posts/taro%E6%BA%90%E7%A0%81%E8%A7%A3%E6%9E%903.5.4/

**Comment by yunfengsa** [comment]
？？
